RecoTracker/TkSeedGenerator/plugins/MultiHitGeneratorFromChi2.cc

```diff
@@ -334,7 +322,7 @@ void MultiHitGeneratorFromChi2::hitSets(const TrackingRegion& region,
     for(int il = 0; (il < size) & (!usePair); il++) {
 
       if (debugPair) 
-	cout << "cosider layer: " << theLayers[il].name() << " for this pair. Location: " << theLayers[il].detLayer()->location() << endl;
+	cout << "cosider layer: " << thirdLayers[il].name() << " for this pair. Location: " << thirdLayers[il].detLayer()->location() << endl;
```

Comment:
Probably yes (also elsewhere in this file where the flag+cout pattern is used), but I'd leave that for another PR if possible.